lib/fizzy/capi.cpp

```diff
@@ -110,13 +110,20 @@ inline fizzy::ExecutionResult unwrap(const FizzyExecutionResult& result) noexcep
         return unwrap(result.value);
 }
 
-inline auto unwrap(FizzyExternalFn func, void* context) noexcept
+inline fizzy::ExecuteFunction unwrap(FizzyExternalFn c_function, void* c_host_context)
 {
-    return [func, context](fizzy::Instance& instance, const fizzy::Value* args,
-               int depth) noexcept -> fizzy::ExecutionResult {
-        const auto result = func(context, wrap(&instance), wrap(args), depth);
-        return unwrap(result);
-    };
+    auto host_context =
+        std::make_any<std::pair<FizzyExternalFn, void*>>(c_function, c_host_context);
+
+    static constexpr fizzy::ExecuteFunctionPtr function =
+        [](std::any host_cntx, fizzy::Instance& instance, const fizzy::Value* args,
```

Comment:
This uses `host_cntx`. I think throughout the project we use `context` or `ctx`, it would be nice to stay consistent.

Reply:
See https://github.com/wasmx/fizzy/pull/716#discussion_r576314628

Reply:
Still, it can named `host_ctx` since the other is `host_context`. But perhaps using the wrapped/unwrapped terms in the context variable name may be more clear.

Reply:
What's the exact suggestion here?

I can actually call them both `host_context` if I move the outer definition after lambda definition and avoid GCC shadowing warning (I did this in some tests)

Reply:
There's an extra `n`. Suggestion is to not use `cntx` but `ctx` or `context` and/or use the `wrapped` or `unwrapped` prefixes.